source/javascripts/_apihandler.js.erb

```diff
@@ -1,11 +1,11 @@
 window.onload = function() {
-	if ("<%= mode %>" == "cli") {
+	if ("<%= mode %>" == "cli" && "<%= build? %>" == "true") {
```

Comment:
@erosdome a suggestion, should not assign a variable to indicate why we are doing this? In the source code it will look sth like this: "if('true' == 'true')". My suggestion:

```javascript
var shouldAbortConnTermination = <%= mode == "CLI" && build ==  %>;
if (shouldCancelApi) { ... }
```

Reply:
Good point, I moved the conditions to an `isPluginMode` variable, somewhat different from your suggestion, but should indicate clearly what scenario it refers to.